### PR TITLE
Create DWMA_deepWhiteMatter.jsonld

### PR DESCRIPTION
**Pull request by lzehl**
For atlas of deep white matter fibre bundles

### CODE REVIEW

**Comment by lzehl** [comment]
@UlrikeS91 if you have time it would be great if you could review these instances (first version) for the Deep White Matter Atlas (which is one of the supported/integrated human EBRAINS atlases).

File: instances/atlas/parcellationEntity/DWMA/DWMA_arcuateFasciculus.jsonld [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "hasParent": [
+      {
+        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/DWMA_deepWhiteMatter"
+      }
+    ],
+    "hasVersion": null,
+    "lookupLabel": "DWMA_arcuateFasciculus",
+    "name": "arcuate fasciculus",
+    "ontologyIdentifier": null,
+    "relatedUBERONTerm": "http://purl.obolibrary.org/obo/UBERON_0035937"
```
Comment (UlrikeS91) [inline instances/atlas/parcellationEntity/DWMA/DWMA_arcuateFasciculus.jsonld]: This needs to point to the controlledTerms/UBERONParcellation instances.

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

**Comment by UlrikeS91** [comment]
I just commented on this one instance, but it of course applies to all ParcellationEntities.

Additionally, ParcellationTerminology(Version) is an embeddedType for BrainAtlas(Version). So, this is wrong too, right? 

If yes, this is in fact wrong for all our atlases....

**Comment by lzehl** [comment]
@UlrikeS91 you are correct. I'm going to fix this asap.

File: instances/atlas/parcellationEntity/DWMA/DWMA_arcuateFasciculus.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/atlas/parcellationEntity/DWMA/DWMA_arcuateFasciculus.jsonld]: done for all

**Comment by lzehl** [comment]
@UlrikeS91 I've updated all BA and BAVs to nest the PaTes and PaTeVs. I've created the missing AMBA parts (at least some). All other changes (e.g. integrationg AtAn into PEVs would touch over 3000 instances at once and I will do this one in another PR)

**Comment by lzehl** [comment]
@UlrikeS91 AtAn merge into PEVs is done via Oli's PR (thanks for the reminder). I just resolved the conflicts and merged it.

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**